### ISSUE TITLE
Test.go file provided in autoscale-go service sample is not working

### ISSUE THREAD

**Issue by awgreene** [knative/docs#412]
Earlier today I attempted to view the autoscaling capabilities of the autoscale-go sample project: https://github.com/knative/docs/blob/master/serving/samples/autoscale-go/README.md

## Expected Behavior
I attempted to run the test.go file located in the test directory with the command line arguments provided at https://github.com/knative/docs/blob/master/serving/samples/autoscale-go/README.md#view-the-autoscaling-capabilities:
```bash
$ go run serving/samples/autoscale-go/test/test.go -sleep 100 -prime 1000000 -bloat 50 -qps 9999 -concurrency 10
```
I had expected the command to maintain 10 Inflight requests, but it quickly fell to 0 or 1.

## Actual Behavior
The test.go file eventually failed to maintain inflight.
```bash
REQUEST STATS:
Total: 85	Inflight: 10	Done: 85 	Success Rate: 100.00%	Avg Latency: 0.1099 sec
Total: 177	Inflight: 10	Done: 92 	Success Rate: 100.00%	Avg Latency: 0.1091 sec
Total: 271	Inflight: 10	Done: 94 	Success Rate: 100.00%	Avg Latency: 0.1070 sec
Total: 365	Inflight: 10	Done: 94 	Success Rate: 100.00%	Avg Latency: 0.1069 sec
Total: 455	Inflight: 10	Done: 90 	Success Rate: 100.00%	Avg Latency: 0.1101 sec
Total: 550	Inflight: 10	Done: 95 	Success Rate: 100.00%	Avg Latency: 0.1064 sec
Total: 639	Inflight: 10	Done: 89 	Success Rate: 100.00%	Avg Latency: 0.1109 sec
Total: 733	Inflight: 10	Done: 94 	Success Rate: 100.00%	Avg Latency: 0.1075 sec
Total: 825	Inflight: 10	Done: 92 	Success Rate: 100.00%	Avg Latency: 0.1072 sec
Total: 916	Inflight: 10	Done: 91 	Success Rate: 100.00%	Avg Latency: 0.1089 sec
Total: 1007	Inflight: 10	Done: 91 	Success Rate: 100.00%	Avg Latency: 0.1101 sec
Total: 10245	Inflight: 1	Done: 9238 	Success Rate: 0.14%	Avg Latency: 0.0002 sec
Total: 20036	Inflight: 1	Done: 9791 	Success Rate: 0.00%	Avg Latency: 0.0000 sec
Total: 29921	Inflight: 1	Done: 9885 	Success Rate: 0.00%	Avg Latency: 0.0000 sec
Total: 39696	Inflight: 1	Done: 9775 	Success Rate: 0.00%	Avg Latency: 0.0000 sec
Total: 49551	Inflight: 1	Done: 9855 	Success Rate: 0.00%	Avg Latency: 0.0000 sec
Total: 59204	Inflight: 0	Done: 9653 	Success Rate: 0.00%	Avg Latency: 0.0000 sec
Total: 68920	Inflight: 0	Done: 9716 	Success Rate: 0.00%	Avg Latency: 0.0000 sec
Total: 78665	Inflight: 0	Done: 9745 	Success Rate: 0.00%	Avg Latency: 0.0000 sec
```

Requests seemingly stopped at a point, so I increased the duration of the test to an hour to see if deployment would scale down to zero, which it did after 5 minutes:
```bash
$ kubectl get deploy --watch 
autoscale-go-00001-deployment   1         1         1         1         45m
autoscale-go-00001-deployment   0         1         1         1         58m
autoscale-go-00001-deployment   0         1         1         1         58m
autoscale-go-00001-deployment   0         0         0         0         58m
```

## Steps to Reproduce the Problem
1. Deploy the autoscale-go sample as shown here: https://github.com/knative/docs/blob/master/serving/samples/autoscale-go/README.md
2. Ramp up traffic to maintain 10 in-flight requests as shown here: https://github.com/knative/docs/blob/master/serving/samples/autoscale-go/README.md#view-the-autoscaling-capabilities

**Comment by josephburnett** [comment]
I think this is fixed by #432.  @awgreene can you try again?

**Comment by samodell** [comment]
Closing. @awgreene , please reopen if necessary.
